lib/sqlitex/query.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Sqlitex.Query do
+  def bind_values({:error, _}=error, _opts), do: error
```

Comment:
Talking to @film42 this morning, he mentioned that this looks a lot like carrying a MaybeMonad through our pipeline. That gave me the idea of using a macro that checks for `{:error, _}` tuples through the pipeline and I found [Elixir Pipes](https://github.com/batate/elixir-pipes#pipe_matching) by Bruce Tate which already does this.

I think I'm going to try rewriting the pipeline one more time with that macro to see if I like the code better